app/controllers/work_packages_controller.rb

```diff
@@ -40,6 +41,8 @@ class WorkPackagesController < ApplicationController
   before_action :load_and_authorize_in_optional_project,
                 :check_allowed_export,
                 :protect_from_unauthorized_export, only: %i[index export_dialog]
+
+  before_action :authorize, only: :show_conflict_flash_message
```

Comment:
🟠  Q: What's the reason behind authorizing only on show conflict? I would presume all controller actions here are to be authorized?

Reply:
This is fine, as `authorize_on_work_package` exists